counsel.el

```diff
@@ -2993,7 +3030,7 @@ NEEDLE is the search string."
 
 ;;;###autoload
 (cl-defun counsel-ag (&optional initial-input initial-directory extra-ag-args ag-prompt
-                      &key caller)
+                                &key caller)
```

Comment:
@abo-abo prefers to use `common-lisp-indent-function`, so the current indentation is intentional.